Some rewording, plus fixes to the CSS and javascript

dpvc opened the pull request:
This PR includes some wording adjustments to the services section.  I wanted to emphasis the accessibility a bit more in places, plus a few other minor things.  I also changed "Beautiful math in all browsers" to "Beautiful and accessible math in all browsers" to push accessibility forward even more.  The simplicity of the original was nice, but I think (and the steering committee suggested) that we needed a stringer accessibility message, so I'm hoping that this helps with that.

I also added the services section into the heading links, adjusted the CSS to switch which sections have the grey background, and included the new link targets in the javascript that manages linking to the pop-ups so they scroll to a better location on screen (it took me a while to figure out that there was such javascript, as I didn't implement that).

Anyway, what you have done looks great, and should help us attract new clients.

dpvc committed:
Some rewording, plus fixes to the CSS and javascript

zorkow commented:
I've seen you removed one of the tracemark signs. Two questions on this:
1. Is there really a trademark on MathJax?
2. Should we remove the sign everywhere?

dpvc commented:
> Is there really a trademark on MathJax?

I don't know.  I've asked the AMS to see.

> Should we remove the sign everywhere?

I had removed it because it was originally only in one place (the first reference to MathJax other than the giant one at the top of the page), so since you had it included in your first one, I removed from its original place.  I've OK not including it, as we certainly don't use it anywhere else in any of the documentation, etc.

zorkow commented:
I've seen the trademark sign somewhere else, too. Let me have another look.
